step-templates/RavenDB - Smuggler - Move Data Between Databases

```diff
@@ -0,0 +1,138 @@
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "ravenSmugglerPath",
+      "Label": "Raven Smuggler Path",
+      "HelpText": "Full path to the Smuggler EXE. \n \nFor example **C:\\Program Files\\Fewzion\\RavenDB\\Smuggler\\Raven.Smuggler.exe**",
+      "DefaultValue": "C:\\Program Files\\Fewzion\\RavenDB\\Smuggler\\Raven.Smuggler.exe",
```

Comment:
Is this an official path to `Raven.Smuggler.exe`? Perhaps it should be left blank if not.